spec/combine/results_combiner_spec.rb

```diff
@@ -101,6 +101,10 @@
       it "has proper results for three.rb" do
         expect(subject[source_fixture("three.rb")]["lines"]).to eq([nil, 3, 7])
       end
+
+      it "always returns a Hash object for branches" do
+        expect(subject[source_fixture("three.rb")]["branches"]).to eq({}) if SimpleCov.branch_coverage_supported?
```

Comment:
You can filter on the spec level: https://relishapp.com/rspec/rspec-core/v/3-10/docs/filtering/conditional-filters

Reply:
TIL, thank you for that! Addressed.